src/BankSystem/Account.java

```diff
@@ -1,13 +0,0 @@
-package BankSystem;
-
```

Comment:
I recommend renaming the package to `banksystem`. as It's a good practice to follow the Java naming conventions. Consider renaming the package from BankSystem to banksystem (all lowercase) to align with the standard conventions.